hips/utils/wcs.py

```diff
@@ -0,0 +1,86 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+import numpy as np
+from astropy.wcs import WCS
+from astropy.coordinates import SkyCoord
+
+__all__ = [
+    'WCSGeometry',
+]
+
+
+class WCSGeometry:
+    """Container for WCS object and image shape.
+
+    Parameters
+    ----------
+    wcs : `~astropy.wcs.WCS`
+        WCS projection object
+    shape : tuple
+        Shape of the image
```

Comment:
Suggest to add an `Examples` section here, showing the example you have in the tests, i.e. how to call `wcs_geo = WCSGeometry.create`, and then print `wcs_geo.wcs` and `wcs_geo.shape` to show it's content.